src/style.css

```diff
@@ -1 +1,64 @@
-/* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+body {
+  margin: 0;
+}
+
+.header {
+  margin: auto;
```

Comment:
your centering doesn't work because the header is a block element and it stretches to the end.